tests/plugins/test_ddp_spawn_plugin.py

```diff
@@ -75,12 +76,43 @@ def test_ddp_spawn_extra_parameters(tmpdir):
     val_name: str = "val_acc"
     model = BoringCallbackDDPSpawnModel(val_name, val)
     dm = BoringDataModule()
-
     trainer.fit(model, datamodule=dm)
     assert trainer.callback_metrics[val_name] == torch.tensor(val)
     assert model.test_val == "test_val"
 
 
+class TestDDPSpawnPlugin(DDPSpawnPlugin):
+    def add_to_queue(self, trainer: Trainer, queue: torch.multiprocessing.SimpleQueue) -> None:
+        queue.put("new_test_val")
+        return super().add_to_queue(trainer, queue)
+
+    def get_from_queue(self, trainer: Trainer, queue: torch.multiprocessing.SimpleQueue) -> None:
+        self.new_test_val = queue.get()
+        return super().get_from_queue(trainer, queue)
+
+
+@RunIf(min_gpus=2)
+def test_ddp_spawn_add_get_queue(tmpdir):
+    """Tests add_to_queue/get_from_queue with DDPSpawnPlugin."""
+
+    ddp_spawn_plugin = TestDDPSpawnPlugin()
+    trainer = Trainer(
+        default_root_dir=tmpdir, fast_dev_run=True, gpus=2, accelerator="ddp_spawn", plugins=[ddp_spawn_plugin]
```

Comment:
```suggestion
        default_root_dir=tmpdir, fast_dev_run=True, num_processes=2, accelerator="ddp_cpu", plugins=[ddp_spawn_plugin]
```

Reply:
@awaelchli I think accelerator needs to be ddp_spawn otherwise we get this test failure:
https://github.com/PyTorchLightning/pytorch-lightning/pull/9118/checks?check_run_id=3563443267

Reply:
these jobs fail for windows + minimal which means it installs torch 1.6. If I remember 1.6 didn't have full support for ddp on windows so that's why we see the error that default group not available. 

So we should add `RunIf(skip_windows=True)` that's probably the best.

Reply:
ahh got it, thank you so much!